tasks/appstudio-utils/Dockerfile

```diff
@@ -2,8 +2,11 @@ FROM registry.access.redhat.com/ubi8/ubi
 
 RUN curl -L https://github.com/mikefarah/yq/releases/download/v4.15.1/yq_linux_amd64 -o /usr/bin/yq && chmod +x /usr/bin/yq
 RUN curl -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.9/openshift-client-linux.tar.gz | tar -xz -C /usr/bin/
+RUN curl -L https://github.com/sigstore/cosign/releases/download/v1.5.1/cosign-linux-amd64 -o /usr/bin/cosign && chmod +x /usr/bin/cosign
+RUN dnf -y copr enable chmouel/tektoncd-cli
```

Comment:
I would rather keep the versioning as for other scripts
```
RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.21.0/tkn-linux-amd64-0.21.0.tar.gz | tar -xz --no-same-owner -C /usr/bin/
```

Reply:
or even better put into dnf install -> https://github.com/tektoncd/cli/releases/download/v0.22.0/tektoncd-cli-0.22.0_Linux-64bit.rpm

Reply:
Good idea. I updated to specify the version.